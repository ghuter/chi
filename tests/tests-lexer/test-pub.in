pub put puub pu_b
