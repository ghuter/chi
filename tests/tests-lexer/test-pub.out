PUB(71)
SPC(8)
IDENTIFIER(5)
<put>
SPC(8)
IDENTIFIER(5)
<puub>
SPC(8)
IDENTIFIER(5)
<pu_b>
EOI(1)
